index.js

```diff
@@ -607,6 +607,15 @@ BabelClient.prototype.error = function error(message) {
     this._log(ERROR, message);
 };
 
+BabelClient.prototype.responseSuccessful = function responseSuccessful(response) {
```

Comment:
should this not be `_responseSuccessful` to try and indicate it's an internal "private" method?

Reply:
I would suggest changing the names would be out of scope, but we should refactor it in the future.